app/frontend/src/assets/scss/style.scss

```diff
@@ -28,47 +45,32 @@ a {
   }
 }
 
+// General Button Style Extensions
+.v-btn {
+  span > span {
```

Comment:
Does this need to be specified to span > span? Or can we just say font weight is bold inside buttons in general?